### PR TITLE
rpg-cli 0.4.1 (new formula)

### PR DESCRIPTION
**Pull request by DNA**
rpg-cli is a bare-bones JRPG-inspired terminal game written in Rust.

It can work as an alternative to cd where you randomly encounter enemies as you change directories.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/rpg-cli.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_equal "rpg-cli #{version}", shell_output("#{bin}/rpg-cli --version").strip
```
Comment (nandahkrishna) [inline Formula/rpg-cli.rb:15]: Can you propose a better test here than checking the version output? Something that tests the functionality of the program is better, as `version` and `help` output comparisons usually pass even for broken installs.

**Review by nandahkrishna — COMMENTED** [review]
Thanks for your PR, @DNA. I have a suggestion. Once you've made your changes, please squash them with your existing commit, and use the same commit message.

File: Formula/rpg-cli.rb (hunk 1 above)
Comment (DNA) [reply Formula/rpg-cli.rb:15]: Hi Nanda! Thanks for the feedback! Let mw know if the new version needs more polishing :)

Comment (nandahkrishna) [reply Formula/rpg-cli.rb]: Seems okay to me 👍🏼.

**Comment by nandahkrishna** [comment]
Seems like there are trailing whitespaces, `brew style --fix rpg-cli` will fix this.

**Review by nandahkrishna — APPROVED** [review]
Thanks for your contribution to Homebrew, @DNA! 🍻

**Pull request closed without merging**